### DISCUSSION TITLE
Training a Dataset with Different People / Animals / ...

### DISCUSSION THREAD

**Discussion by m-w-marker** [d8ahazard/sd_dreambooth_extension#1223] [Q&A]
Hi,

does someone know how to train a Dataset with different People?
I have a Dataset of People with different Animal Heads, but I have no Idea how to Class or Instance Prompt it.
That means - someone has a Zebra or a Lion or a Monkey Head and so on.

My Instance token is ANHE 
My Class token is Animal

But this makes no sense because I have different Animals, and the training gives me just bad results.

Thx :)

**Comment by kohweiteck** [comment]
My suggestion is to reframe the concept class to "People" and your captioning prompt should "People wearing ANHE <animal> head mask "
